src/main/java/Models/User/User.java

```diff
@@ -21,15 +22,15 @@
 //     private Date dob;
 //
 //     private String gender;
```

Comment:
Do we actually plan to include the above commented values?

Reply:
Currently I am unsure - they are not specifically required, so can be removed in the repo if we like.

Reply:
Perhaps we can discuss this in our next standup.